app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -25,7 +25,7 @@ def initialize(ems, options = nil)
       @network_service            = @os_handle.detect_network_service
       @image_service              = @os_handle.detect_image_service
       @volume_service             = @os_handle.detect_volume_service
-      @storage_service            = @os_handle.detect_storage_service
+      @storage_service            = @os_handle.detect_storage_service("admin")
```

Comment:
From what I can see, this is temporary and actually gets removed by #3278, in which the pagination handler actually handles the error conditions from queries.